Purchases/Public/RCPurchases.m

```diff
@@ -351,10 +351,10 @@ + (void)addAttributionData:(NSDictionary *)data
                fromNetwork:(RCAttributionNetwork)network
           forNetworkUserId:(nullable NSString *)networkUserId {
     if (_sharedPurchases) {
-        RCLog(@"There is an instance configured, posting attribution.");
+        RCLog(RCStrings.attribution.instance_configured_posting_attribution);
         [_sharedPurchases postAttributionData:data fromNetwork:network forNetworkUserId:networkUserId];
     } else {
-        RCLog(@"There is no instance configured, caching attribution.");
+        RCLog(RCStrings.attribution.no_instance_configured_caching_attribution);
```

Comment:
Here's the sample usage. My thinking was to have one namespace per Group as defined in the spreadsheet.
We have to move `RCLog` to Swift so that it can be used everywhere, I'm doing that on a separate PR.